packages/app/src/app/overmind/namespaces/comments/actions.ts

```diff
@@ -102,7 +102,7 @@ export const closeComment: Action = ({ state }) => {
 
 export const selectComment: AsyncAction<{
   commentId: string;
-  bounds: {
+  bounds?: {
```

Comment:
```suggestion
  bounds: {
```

Since we now always pass it 😄